implementations/rust/ockam/ockam_transport_ble/examples/04-routing-over-ble-transport-initiator.rs

```diff
@@ -1,12 +1,21 @@
 // This node routes a message, to a worker on a different node, over the ble transport.
 
-use ockam::{route, Context, Result};
-use ockam_transport_ble::{BleClient, BleTransport, BLE};
+use ockam_core::{route, Result};
+use ockam_node::Context;
 
 use ockam_transport_ble::driver::btleplug::BleAdapter;
+use ockam_transport_ble::{BleClient, BleTransport, BLE};
+
+fn main() -> Result<()> {
+    let (ctx, mut exe) = ockam_node::start_node();
+    exe.execute(async move {
+        async_main(ctx).await.unwrap();
+    })
+    .unwrap();
+    Ok(())
+}
 
-#[ockam::node]
```

Comment:
Can we make it work without ockam dependency? @adrianbenavides 
Should not prevent this PR from landing though

Reply:
That's been dropped here - I just start up the context and executor manually rather than pulling all the macro machinery in:

https://github.com/ockam-network/ockam/blob/615851692f4c61f910a030dc04421a86c56b312d/implementations/rust/ockam/ockam_transport_ble/examples/04-routing-over-ble-transport-initiator.rs#L9-L20

Reply:
Yes, we have two options:
1. `pub use` the node macro at [ockam_core](https://github.com/ockam-network/ockam/blob/develop/implementations/rust/ockam/ockam_core/src/lib.rs#L47) and use the macro as `#[ockam_core::node(crate = "ockam_node")]` or similar
2. explicitly depend on `ockam_macros` and use the macro as `#[ockam_macros::node(crate = "ockam_node")]`